content/200-concepts/100-components/01-prisma-schema/06-relations/index.mdx

```diff
@@ -281,54 +265,50 @@ model Category {
+</Admonition>
 
-If you're not using Prisma Migrate but obtain your data model from [introspection](/concepts/components/introspection), you can still make use of implicit many-to-many relations by following Prisma's [conventions for relation tables](many-to-many-relations#conventions-for-relation-tables-in-implicit-m-n-relations).
+Notice that the syntax is slightly different between relational databases and MongoDB - particularly for [many-to-many relations](many-to-many-relations).
 
-For **relational databases**, the following entity relationship diagram represents the database that corresponds to the sample Prisma schema:
+For relational databases, the following entity relationship diagram represents the database that corresponds to the sample Prisma schema:
 
 ![The sample schema as an entity relationship diagram](sample-schema.png)
 
-For **MongoDB**, Prisma uses a [normalized data model design](https://docs.mongodb.com/manual/core/data-model-design/), which means that documents reference each other by ID in a similar way to relational databases:
+For MongoDB, Prisma uses a [normalized data model design](https://docs.mongodb.com/manual/core/data-model-design/), which means that documents reference each other by ID in a similar way to relational databases. See [the MongoDB section](#mongodb) for more details.
 
-For example, the following MongoDB document represents a `User`:
+### Implicit and explicit many-to-many relations
 
-```json
-{ "_id": { "$oid": "60d5922d00581b8f0062e3a8" }, "name": "Ella" }
-```
+Many-to-many relations in relational databases can be [explicit](/concepts/components/prisma-schema/relations/many-to-many-relations#explicit-many-to-many-relations), where the relation table is represented as an explicit model in your Prisma schema, or [implicit](/concepts/components/prisma-schema/relations/many-to-many-relations#implicit-many-to-many-relations), where Prisma manages the relation table and it does not appear in the Prisma schema.
+
+Implicit many-to-many relations require both models to have a single `@id`. Be aware that:
 
-The following list of `Post` documents each have a `userId` field which reference the same user:
+- You cannot use a [multi-field ID](/reference/api-reference/prisma-schema-reference#id-1)
+- You cannot use a `@unique` in place of an `@id`
 
-```json
-[
-  {
-    "_id": { "$oid": "60d5922e00581b8f0062e3a9" },
-    "title": "How to make sushi",
-    "authorId": { "$oid": "60d5922d00581b8f0062e3a8" }
-  },
-  {
-    "_id": { "$oid": "60d5922e00581b8f0062e3aa" },
-    "title": "How to re-install Windows",
-    "authorId": { "$oid": "60d5922d00581b8f0062e3a8" }
-  }
-]
-```
+To use either of these features, you must set up an explicit many-to-many instead.
+
+The implicit many-to-many relation still manifests in a relation table in the underlying database. However, this relation table is managed by Prisma.
```

Comment:
```suggestion
The implicit many-to-many relation still manifests in a relation table in the underlying database. However, Prisma manages this relation table.
```

To avoid passive voice (but as a piece of writing, I prefer what you wrote!)